include/grpc++/support/sync_stream.h

```diff
@@ -151,17 +165,21 @@ class ClientReader GRPC_FINAL : public ClientReaderInterface<R> {
   Call call_;
 };
 
+/// Client-side interface for streaming writes of message of type \a W.
 template <class W>
 class ClientWriterInterface : public ClientStreamingInterface,
                               public WriterInterface<W> {
  public:
+  /// Block until writes are completed.
```

Comment:
This does not sound right. WritesDone indicates the client has no more message to send